system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -43,6 +43,7 @@ function countTopSitesIconsTypes(topSites) {
 export class _TopSites extends React.PureComponent {
   constructor(props) {
     super(props);
+    this.windowObj = this.props.windowObj || window;
```

Comment:
I believe we should be using `global` and overriding it in tests. (for prerendering)

Reply:
I tried that but `window` is a getter-only property for globals. I can do `global.window.innerWidth = 1056;` in the tests and it works but that seems hacky (?).